03-command-line-todo-list/lib/cli.rb

```diff
@@ -0,0 +1,73 @@
+require "thor"
+require_relative "database"
+
+module Todo
+  class CLI < Thor
+    def initialize(*args)
+      @store = Todo::Database.new
+
+      super
+    end
+
+    desc "-a or --add [TASK]", "Adds an item into the list of tasks"
+    map %w[-a --add] => :add
+    def add(task)
+      @store.tasks = @store.tasks << task
+
+      pretty_print_tasks(@store.tasks)
+    end
+
+    desc "-d or --delete [NUMBER]", "Deletes task at index NUMBER from the list"
+    map %w[-d --delete] => :delete
+    def delete(item)
+      tasks = @store.tasks
+
+      if tasks.length < item.to_i
+          shell.say "Task at number #{item} does not exist\n\n"
+          return
+      end
+
+      shell.say "\n\"\e[1m#{tasks.delete_at(item.to_i - 1)}\"\e[0m [removed]\n\n"
```

Comment:
Probably should separate the business logic from showing messages to user to make the presentation logic less invasive

Reply:
Also could be cool to provide some kind of helper that removes the noise from using color codes